src/mediapipe_internal/mediapipegraphconfig.hpp

```diff
@@ -23,6 +23,9 @@
 
 namespace ovms {
 
+const std::string DEFAULT_GRAPH_FILENAME = "graph.pbtxt";
```

Comment:
this way we have redefinitions whenever this is included in some cpp.
to avoid that, keep definition in mediapipegraphconfig.cpp and use extern in header file.